src/api/Metadata.js

```diff
@@ -407,17 +501,32 @@ class Metadata extends File {
                 hasMetadataFeature ? this.getTemplates(id, METADATA_SCOPE_ENTERPRISE) : Promise.resolve([]),
             ]);
 
-            const editors = await this.getEditors(
-                id,
-                instances,
-                customPropertiesTemplate,
-                enterpriseTemplates,
-                globalTemplates,
-                !!permissions.can_upload,
-            );
+            // Filter out classification
+            const filteredInstances = this.extractClassification(id, instances);
+
+            const templateInstances = isMetadataRedesign
+                ? await this.getTemplateInstances(
+                      id,
+                      filteredInstances,
+                      customPropertiesTemplate,
+                      enterpriseTemplates,
+                      globalTemplates,
+                  )
+                : [];
+            const editors = !isMetadataRedesign
+                ? await this.getEditors(
+                      id,
+                      filteredInstances,
+                      customPropertiesTemplate,
+                      enterpriseTemplates,
+                      globalTemplates,
+                      !!permissions.can_upload,
```

Comment:
is there an equivalent check to `!!permissions.can_upload` needed for templateInstances?

Reply:
I analysed the implementation and yes, this will be needed, thanks :) I added this prop